Makefile

```diff
@@ -15,4 +15,8 @@ install:
 
 .bin/ory: Makefile
 	bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.38
-	touch -a -m .bin/ory
\ No newline at end of file
+	touch -a -m .bin/ory
+
+.PHONY: format
```

Comment:
No need to make this target `.PHONY` since there is no `format` file or folder in this codebase.

```suggestion
```

Reply:
Done.